blocks/init/src/Blocks/wrapper/components/wrapper-options.js

```diff
@@ -1,369 +1,272 @@
-															allowReset={true}
-															value={attributes[attr]}
-															onChange={(value) => setAttributes({ [attr]: value })}
-															min={options.wrapperWidth.min}
-															max={options.wrapperWidth.max}
-															step={options.wrapperWidth.step}
-															resetFallbackValue={reset[attr]?.default}
-														/>
-													</Fragment>
-												);
-											})}
-										</Responsive>
-									}
-
-									{showWrapperContainerWidth &&
-										<Responsive label={<IconLabel icon={icons.containerWidth} label={__('Container width', 'eightshift-frontend-libs')} />}>
-											{options.breakpoints.map((item, index) => {
-												const point = ucfirst(item);
-												const attr = `wrapperContainerWidth${point}`;
-
-												return (
-													<Fragment key={index}>
-														<CustomSelect
-															label={<IconLabel icon={icons[`screen${point}`]} label={point} />}
-															options={options.wrapperContainerWidth}
-															value={attributes[attr]}
-															onChange={(value) => setAttributes({ [attr]: value })}
-															isSearchable={false}
-															isClearable={false}
-															simpleValue
-														/>
-													</Fragment>
-												);
-											})}
-										</Responsive>
-									}
-
-									{showWrapperGutter &&
-										<Responsive label={<IconLabel icon={icons.gutter} label={__('Gutter', 'eightshift-frontend-libs')} />}>
-											{options.breakpoints.map((item, index) => {
-
-												const point = ucfirst(item);
-												const attr = `wrapperGutter${point}`;
-
-												return (
-													<Fragment key={index}>
-														<CustomSelect
-															label={<IconLabel icon={icons[`screen${point}`]} label={point} />}
-															options={options.wrapperGutter}
-															value={attributes[attr]}
-															onChange={(value) => setAttributes({ [attr]: value })}
-															isSearchable={false}
-															isClearable={false}
-															simpleValue
-														/>
-													</Fragment>
-												);
-											})}
-										</Responsive>
-									}
-								</>
-							}
-
-							{showWrapperAnchorId &&
-								<TextControl
-									label={<IconLabel icon={icons.anchor} label={__('Block anchor ID', 'eightshift-frontend-libs')} />}
-									value={wrapperAnchorId}
-									onChange={(value) => setAttributes({ wrapperAnchorId: value })}
-								/>
-							}
+					{wrapperSpacingBottomInShow &&
+						<Responsive label={<IconLabel icon={icons.spacingBottomIn} label={__('Bottom inner spacing', 'eightshift-frontend-libs')} />}>
+							{options.breakpoints.map((item, index) => {
+								const point = ucfirst(item);
+								const attr = `wrapperSpacingBottomIn${point}`;
+								const { min, max, step } = getOption('wrapperSpacingInner', attributes, manifest);
```

Comment:
Why don't you extract this outside of the loop other than calling it each time? (extract all the get options somewhere up?)